specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/bigDataPool.json

```diff
@@ -568,12 +572,20 @@
         "minExecutors": {
           "type": "integer",
           "format": "int32",
-          "description": "The minimum number of executors alloted"
+          "description": "The minimum number of executors alloted",
+          "x-ms-mutability": [
```

Comment:
Is it intentional that this field couldn't read?

refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-mutability